src/core/Akka.Remote.TestKit/CommandLine.cs

```diff
@@ -43,10 +43,20 @@ public static string GetProperty(string key)
             return Values.Value[key];
         }
 
+        public static string GetPropertyOrDefault(string key, string defaultStr)
```

Comment:
Added some overloads to `CommandLine` to make it easier to specify defaults for optional parameters.